.cirrus.yml

```diff
@@ -807,7 +807,59 @@ success_task:
         CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
         TEST_ENVIRON: container
     clone_script: *noop
-    script: /bin/true
+    script: *noop
+
+
+artifacts_task:
+    name: "Artifacts"
+    alias: artifacts
+    only_if: *not_docs
+    depends_on:
+        - success
+    # This task is a secondary/convenience for downstream consumers, don't
+    # block development progress if there is a failure in a PR, only break
+    # when running on branches or tags.
+    allow_failures: $CIRRUS_PR != ''
+    container: *smallcontainer
+    env:
+        CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+        TEST_ENVIRON: container
+        CURL: "curl --fail --location -O https://api.cirrus-ci.com/v1/artifact/build/${CIRRUS_BUILD_ID}"
+    # In order to keep the download URL and Cirrus-CI artifact.zip contents
+    # simple, nothing should exist in $CIRRUS_WORKING_DIR except for artifacts.
+    clone_script: *noop
+    script:
+        # Assume the latest Fedora release build is most useful
+        - $CURL/Build%20for%20$FEDORA_NAME/binary/bin/podman
+        - $CURL/Build%20for%20$FEDORA_NAME/binary/bin/podman-remote
+        - $CURL/Build%20for%20$FEDORA_NAME/binary/bin/rootlessport
+        - chmod +x podman* rootlessport
+        # Architecture in filename & can't use wildcards in a URL
+        - mkdir -p /tmp/alt
+        - cd /tmp/alt
```

Comment:
Previous iteration had these as subdirectories of `$CIRRUS_WORKING_DIR`, a directory which (assuming line 829 is correct) is empty. Do we have such guarantees about `/tmp/alt` and `/tmp/win`? I kind of prefer the all-under-one-subdirectory approach.

Reply:
Yep, it's a fresh VM image at boot so `/tmp/` will be empty except for the usual systemd stuff.